### PR TITLE
PAYARA-1415 Added test notifiers admin console and command

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
jenkins test please

**Comment by Cousjava** [comment]
Not tested SNMP or Slack notifier working

**Comment by Cousjava** [comment]
jenkins test please

File: appserver/payara-appserver-modules/notification-snmp-core/src/main/java/fish/payara/notification/snmp/SnmpNotificationRunnable.java [hunk 1]
```diff
@@ -95,6 +95,7 @@ public void run() {
                 pdu.add(new VariableBinding(oidInstance, new OctetString(message.getSubject() + "\n" + message.getMessage())));
 
                 snmp.send(pdu, communityTarget);
+                logger.log(Level.FINE, "Message sent seuccessfully");
```
Comment (michaelranaldo) [inline appserver/payara-appserver-modules/notification-snmp-core/src/main/java/fish/payara/notification/snmp/SnmpNotificationRunnable.java]: seuccessfully -> successfully

File: appserver/admingui/datadog-notifier-console-plugin/src/main/resources/fish/payara/admingui/notifier/datadog/Strings.properties [hunk 2]
```diff
@@ -43,4 +43,6 @@ notifier.datadog.configuration.pageHelpText=Enable and configure a Datadog notif
 notifier.datadog.configuration.enabledLabel=Enabled
 notifier.datadog.configuration.enabledLabelHelpText=Enables/Disables the Datadog notifier
 notifier.datadog.configuration.keyLabel=Key
-notifier.datadog.configuration.keyLabelHelpText=Datadog key
\ No newline at end of file
+notifier.datadog.configuration.keyLabelHelpText=Datadog key
```
Comment (michaelranaldo) [inline appserver/admingui/datadog-notifier-console-plugin/src/main/resources/fish/payara/admingui/notifier/datadog/Strings.properties]: key -> Key

**Comment by Cousjava** [comment]
jenkins test please

**Comment by Cousjava** [comment]
jenkins test please

**Comment by Cousjava** [comment]
jenkins test please

**Comment by michaelranaldo** [comment]
Jenkins test please

**Comment by mulderbaba** [comment]
jenkins test please

**Comment by Cousjava** [comment]
Jenkins, would you kindly?

**Comment by Cousjava** [comment]
jenkins test please

**Comment by Cousjava** [comment]
@lprimak No, it can't be refactored as each time it is using different classes that all work in slightly different ways.

**Comment by Cousjava** [comment]
jenkins test please

**Pull request merged**